src/main/java/hudson/plugins/translation/L10nDecorator.java

```diff
@@ -99,7 +101,7 @@ public String encodeRecording(StaplerRequest request) throws IOException {
     public List<Msg> decode(StaplerRequest request) throws IOException {
         BufferedReader r = new BufferedReader(new InputStreamReader(new GZIPInputStream(new CipherInputStream(
                 new ByteArrayInputStream(Base64.decode(request.getParameter("bundles").toCharArray())),
-                getCipher(DECRYPT_MODE)))));
+                getCipher(DECRYPT_MODE))), StandardCharsets.UTF_8));
```

Comment:
The core version of Jenkins for this plugin is 1.424 so as currently stands that needs to be able to run on Java 1.5